pyrefly.toml

```diff
@@ -31,6 +31,7 @@ project-includes = [
     "docs/helpers.py",
     "examples/converters/convert_dcp_to_hf.py",
     "examples/converters/convert_megatron_to_hf.py",
+    "examples/searchR1/data/searchr1_download.py",
```

Comment:
I'm wondering, why particularly `searchr1_download.py` but not the other python files?